storybook/cypress/support/e2e.ts

```diff
@@ -18,3 +18,13 @@ import "./commands";
 
 import "cypress-axe";
 import "cypress-real-events";
+
+import { addMatchImageSnapshotCommand } from "@simonsmith/cypress-image-snapshot/command";
+
+addMatchImageSnapshotCommand({
+  blur: 2,
```

Comment:
Waar is deze voor nodig?

Ik heb https://github.com/americanexpress/jest-image-snapshot#%EF%B8%8F-api nagelezen maar ik begrijp de toepassing nog niet precies.

Reply:
Zoals ik het interpreteer zorgt `blur` ervoor dat de screenshot minder precies wordt en minimale pixel verschillen erdoor worden 'uitgewist'. 

zonder blur zijn er ... verschillen
met blur: 1 zijn er ... verschillen
met blur: 2 zijn er geen verschillen meer

De verschillen die hierdoor worden geëlimineerd zijn:

[Nog bijwerken]

Reply:
Opgelost door `thresholdLimitType: "percent"` te zetten